### ISSUE TITLE
[Major Bug] Restarting Co-op game (mid-game) causes all clients game's to freeze

### ISSUE THREAD

**Issue by Master-64** [shiversoftdev/t7patch#11]
As the title says. The freeze is permanent across all clients upon restarting the game (through /map_restart), except for the host, who can continue to play as normal.

**Comment by shiversoftdev** [comment]
Probably related to handler dropping campaign related lobby message data. Will todo this

**Comment by shiversoftdev** [comment]
Please verify if this issue still exists in 1.06. Thanks!

**Comment by Master-64** [comment]
Will check soon.

**Comment by Master-64** [comment]
Can confirm that this is still an issue on v1.06. Result is identical to that of v1.05.

**Comment by shiversoftdev** [comment]
Alright, well, that means the bug is either not what I thought it was or I might have missed something small. Ill try to test a proper fix before 1.07.

**Comment by clx127** [comment]
Yeah... Me too on custom mp game, randomly freeze game:(

**Comment by shiversoftdev** [comment]
If you would like this to be fixed please send a crashes.log from one of the clients. Thank you!

**Comment by Master-64** [comment]
Here's the crash log that the client received upon the host (me) running the console command "/map_restart": https://gist.github.com/Master-64/11c39057e3fbc37a9802addf6a1c94ea

Also worth mentioning that since using v1.10, the same client randomly crashed mid-game, something I've never seen before (this never occurred on v1.06). Here's the crash log for that: https://gist.github.com/Master-64/96fae6ba2e0df981c65b7b0dfa2b30d6

**Comment by Master-64** [comment]
Can confirm that this issue still exists on v1.12 of this patch, despite it stating it was fixed.

**Comment by shiversoftdev** [comment]
I never once stated that it was fixed... don't know where you got this from. This is just a non-issue for me right now and Ill fix it when I have time.

**Comment by Master-64** [comment]
Oh, I'm stupid, I read it as being fixed for some reason when it actually said known issue. Sorry for the confusion.

**Comment by Brandon98SE** [comment]
Me and my friend were playing split-screen while connected to my brother through steam and we had different maps crash like 3 times on us and we decided to just play without the program but to play on a closed lobby, I know probably not safe but my windows installation is very locked down anyway so it should be ok, We played like that for ages and didn't have a single crash..

tldr: Program causes crash when playing split-screen, I can send logs if needed.

**Comment by shiversoftdev** [comment]
Finally think I know why this happens and will test the fix in 1.13

**Comment by shiversoftdev** [comment]
Please retest and let me know if this issue is fixed. Thanks!

**Comment by Brandon98SE** [comment]
I confirm that this bug has been fixed, my Mum and I were playing Black Ops 3 Zombies last night (splitscreen) with a friend (on his own machine) for over 2 hours and we had no such issues with T7Patch v2.0 running.

**Comment by Master-64** [comment]
Fixed
